pilot/cmd/pilot-agent/status/server.go

```diff
@@ -225,10 +228,10 @@ func (s *Server) handleAppProbe(w http.ResponseWriter, req *http.Request) {
 		},
```

Comment:
you want to change line 223 as well, right? the request flow is kubelet -> pilot agent -> application.

your pr make change on the first part, but the second part is still hardcoded 10 second.  if a probing request takes 20 seconds to finish within application, the pilot agent will false negatively claim the request fails too early.

Reply:
and you should also add a test in server_test.go to simulate a test app within process, handling a particular type of request very long, and using a prober with correct prober timeout config. and you shall see the test succeed only after your PR